cob_twist_controller/src/limiters/limiter.cpp

```diff
@@ -288,13 +288,13 @@ KDL::JntArray LimiterIndividualJointVelocities::enforceLimits(const KDL::JntArra
 
     uint16_t maxDof = this->tc_params_.dof;
     std::vector<double> tmpLimits = this->tc_params_.limits_vel;
-    if(this->tc_params_.base_active)
-    {
-        maxDof += 3; // additional 3 DOF for the base (X, Y, Z)
-        tmpLimits.push_back(this->tc_params_.max_vel_lin_base); // BaseTransX limit
-        tmpLimits.push_back(this->tc_params_.max_vel_lin_base); // BaseTransY limit
-        tmpLimits.push_back(this->tc_params_.max_vel_rot_base); // BaseRotZ limit
-    }
+    //if(this->tc_params_.base_active)
+    //{
+        //maxDof += 3; // additional 3 DOF for the base (X, Y, Z)
+        //tmpLimits.push_back(this->tc_params_.max_vel_lin_base); // BaseTransX limit
+        //tmpLimits.push_back(this->tc_params_.max_vel_lin_base); // BaseTransY limit
+        //tmpLimits.push_back(this->tc_params_.max_vel_rot_base); // BaseRotZ limit
+    //}
```

Comment:
apply limiters in inverse_differential_kinematics_solver rather than twist_controller....then this can be added back in...